docs/rules/switch-final-break.md

```diff
@@ -0,0 +1,88 @@
+# Checks whether the final clause of a switch statement ends in unnecessary `break;`. (switch-final-break)
```

Comment:
This should match what we have in the rule's `meta.description`: https://github.com/eslint/eslint/pull/12094/files#diff-cff99b58163f559f91380878b55965d9R21-R22